### DISCUSSION TITLE
Use Cases and General Audience

### DISCUSSION THREAD

**Discussion by briandowns** [Jason2605/Dictu#499] [Q&A]
I was hoping I might gain some clarity on who the intended audiences are for this language and some use cases where Dictu has been used successfully.

**Comment by Jason2605** [comment]
It's a very good question and one that is quite hard to answer! This language initially started out as more of a hobbyist project for myself and ended up evolving into something a little more than that; so when starting out, I didn't really have an intended target or niche I wanted to capitalise on. This obviously isn't great for language adoption as there are plenty of other high-level scripting languages out there with a much larger eco-system.

As for projects out there in the wild that are using Dictu, the only public facing one that I have (which is pretty barebones) is this: https://paste.dictu-lang.com/ (It's essentially a pastebin clone.)

That said, I would definitely not be opposed to attempting to create more public facing projects powered by Dictu 😄
